objc/notify.m

```diff
@@ -58,9 +58,16 @@ BOOL setApplication(NSString* newbundleIdentifier)
         userNotification.informativeText = message;
 
         // Notification sound
-        if (options[@"sound"] && ![options[@"sound"] isEqualToString:@""] && ![options[@"sound"] isEqualToString:@"_mute"])
+        if (options[@"sound"] && ![options[@"sound"] isEqualToString:@""])
         {
-            userNotification.soundName = options[@"sound"];
+            if (![options[@"sound"] isEqualToString:@"NSUserNotificationDefaultSoundName"])
```

Comment:
Not sure i find `NSUserNotificationDefaultSoundName` to be best suited for a `default` sound option.
I propose a enum instead:

```
enum Sound {
  Default,
  Custom(String)
}
```

Reply:
this will probably require a breaking change so I will let it up to the maintainers whether they want this or not

Reply:
I think this breaking change is acceptable in this case, there are going to be a few more of these before the next release anyway, so please go break things :D